template2.html

```diff
@@ -0,0 +1,180 @@
+                 </p>
+            </div>
+            <footer>
+               <div id="skills_2" class="skills">
+                 <p>html</p>
+                 <p>css</p>
+                 <p>javaScript</p>
+               </div>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+              </footer>
+  
+            
+          </section>
+          <section class="p">
+            <div>
+                  <img  class="project_image" src="statics/images/Snapshoot_Portfolio3.png" alt="snapshoot image project">
+                  <h1>Multi-Post
+                    Stories</h1>
+                  <div id="general_inf_3">
+                      <p class="title">CANOPY</p>
+                      <img class="counter" src="statics/images/Counter.png" alt="counter"/>
+                      <p class="job-title">Back End Dev</p>
+                      <img class="counter" src="statics/images/Counter.png" alt="counter">
+                      <p class="year">2015</p>
+                  </div>
+              
+                 <p class="project_description">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+                 </p>
+            </div>
+            <footer>
+               <div id="skills_3" class="skills">
+                 <p>html</p>
+                 <p>css</p>
+                 <p>javaScript</p>
+               </div>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+              </footer>
+  
+            
+          </section>
+          <section id="about_me">
+              <h1>About<br> Myself</h1>
+              <p id="presentation">Hello I’m a software developer! I can help you build a product , 
+                  feature or website Look through some of my work and experience!
+                   If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+              <div id="profile_link">
+                <h2>LET’S CONNECT</h2>
+                <ul>
+                    <li><a href="#"><img src="statics/images/github.png" alt="github profilelink"></a></li>
+                    <li><a href="#"><img src="statics/images/Linkedin_icon.png" alt="linkedin profile link"></a></li>
+                    <li><a href="#"><img src="statics/images/doits.png" alt="doit"></a></li>
+                    <li><a href="#"><img src="statics/images/twitter.png" alt="twitter profile link"></a></li>
+                    <li><a href="#"><img src="statics/images/M.png" alt="M link"></a></li>
+                </ul>
+                <a id="my_resume" href="#">Get my resume</a> 
+              </div>
+              <div id="my_skills">
+                  <div id="languages_skills"> 
+                      <div id="spec_name">
+                          <p>Langguages</p>
```

Comment:
- [ ] Here is a typo, please consider fixing it in order to have a more professional project.